src/controllers/user.js

```diff
@@ -82,7 +80,13 @@ export const updateById = async (req, res) => {
         message: 'Cohort could not be found'
       })
     }
-    return sendDataResponse(res, 200, { test: updatedProfile })
+
+    const updatedProfile = await userToUpdate.update()
+    userToUpdate.cohortId = cohortId
+
+    return sendDataResponse(res, 200, {
+      user: { cohort_id: updatedProfile.cohortId || null }
```

Comment:
What's the reasoning for this?

Was this to avoid the problem of sending across a string ID rather than a number? If so - i'd expect an error message, saying the cohort wasn't found

Reply:
I've removed || null